test/e2e/tests/frontend/rsvp.spec.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require('@wordpress/e2e-test-utils-playwright');
+
+test.describe('RSVP to an event', () => {
+	/* 
+	test('02-verify the non-logged in user view RSVP button on home page and perform RSVP action', async ({
+		page,
+	}) => {
+		await page.goto('https://develop.gatherpress.org');
+		await page.getByRole('heading', { name: 'Upcoming Events' }).isVisible();
+		await page
+			.locator('div')
+			.filter({ hasText: /^online Test Event$/ })
+			.isVisible();
+	
+		await page.getByRole('link', { name: 'RSVP' }).first().isVisible();
+	
+		await page.getByRole('link', { name: 'RSVP' }).first().click();
+		await page.getByText('Login', { exact: true }).click();
+	
+		await loginUser({ page, username: 'testuser1' });
+		await page.evaluate(() => window.scrollTo(0, 1000));
+	
+		await page
+			.getByRole('link', { name: 'RSVP' })
+			.first()
+			.click({ timeout: 60000 });
+	
+		await page.locator('a').filter({ hasText: 'Attend' }).click();
+		await page.getByText('Close').click();
+		await page.locator('.gatherpress-rsvp-response__items').first().isVisible(); // verified the RSVP button is visible
+		await expect(page.getByText('Attending').first()).toBeVisible(); // verified the attending text after RSVP action.
+	
+		await page.locator('.gatherpress-rsvp-response__items').first().isVisible(); // verified the attending users list.
+		await page
+			.locator('.gatherpress-rsvp-response__items')
+			.first()
+			.screenshot({ path: 'attending.png' });
+	}); */
+
+	test('A logged in user can perform RSVP action', async ({ page }) => {
+		await page.goto('/');
+		await page.getByRole('link', { name: 'Events' }).click({ force: true }); // Imported from of https://github.com/GatherPress/gatherpress-demo-data
+
+		await page.evaluate(() => window.scrollTo(0, 5000));
+		await page
+			.getByRole('link', { name: 'RSVP' })
+			.first()
+			.click({ force: true, timeout: 60000 });
+
+		await page
+			.locator('a')
+			.filter({ hasText: 'Attend' })
+			.click({ force: true });
+		await page.getByText('Close').click({ force: true });
```

Comment:
Forcing clicks should be used only in [specific edge cases](https://playwright.dev/docs/input#forcing-the-click:~:text=Sometimes%2C%20apps%20use%20non%2Dtrivial%20logic%20where%20hovering%20the%20element%20overlays%20it%20with%20another%20element%20that%20intercepts%20the%20click.). Do we know if it's necessary to use it here? I'm asking because it bypasses the auto-waiting mechanism which can lead to flaky results.

Reply:
Hello again @WunderBart, thanks for looking at this!

Yes, I'm aware of this bad behavior, as you can see from some of the inline comments and my commit messages like [TRYOUT .click({ force: true }) which is ugly, but works :(](https://github.com/GatherPress/gatherpress/pull/810/commits/e405a0f8af711447f0142072e33eed35db56a470).

I experienced, that using `force:true` helped avoiding the before mentioned timeouts (error 500) to a minimum or even to not happen at all. With the commits from yesterday, I was a least able to reduce the failing tests to one, followed by an error 500. But it was only one 500 anymore, not 15-20 like in the workflow runs before.

But yes - all in all - I would like to remove all forcing too, if I (or maybe we) find a way around the fatal errors.